### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by robertjd**
See changelog

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -6,6 +6,31 @@ Change Log
 
 All library changes, in descending order.
 
+Version 2.4.0
+-------------
+
+**Attention: Minor release the affects ID Site and Social Users**
```
Comment (timothyej) [inline docs/changelog.rst]: Typo:

> Minor release **the** affects

`the` => `that`

File: docs/upgrading.rst [hunk 2]
```diff
@@ -7,12 +7,30 @@ Upgrade Guide
 This page contains specific upgrading instructions to help you migrate between
 Express-Stormpath releases.
 
+Version 2.3.7 -> Version 2.4.0
+------------------------------
+
+**Attention: Minor release the affects ID Site and Social Users**
```
Comment (timothyej) [inline docs/upgrading.rst]: Typo:

> Minor release **the** affects

`the` => `that`

File: docs/changelog.rst [hunk 3]
```diff
@@ -6,6 +6,31 @@ Change Log
...
+
+**Released February 8, 2016.**
+
+We have improved security and consistency for our ID Site and Social
+integrations.  These integrations now receive the same access token and refresh
+cookies that you see when logging in with password-based authentication.
```
Comment (timothyej) [inline docs/changelog.rst]: > These integrations now receive the same access token and **refresh cookies**...

`refresh cookies` and not `refresh tokens`?

**Comment by timothyej** [comment]
Did you forgot to change the version in `package.json`?

**Comment by robertjd** [comment]
Yeah @timothyej I forgot package.json :) other edits made, thanks!

**Pull request merged**